### PR TITLE
VolumeSubpathEnvExpansion feature gate is enabled by default from v1.15

### PR DESCRIPTION
**Pull request by houjun41544**
VolumeSubpathEnvExpansion feature gate is enabled by default from v1.15

### CODE REVIEW

File: content/en/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -1158,7 +1158,7 @@ spec:
 
 
 Use the `subPathExpr` field to construct `subPath` directory names from Downward API environment variables.
-Before you use this feature, you must enable the `VolumeSubpathEnvExpansion` feature gate.
+This feature is enabled by default.
```
Comment (sftim) [inline content/en/docs/concepts/storage/volumes.md]: Readers might still want to know the name of the feature gate that controls subPath expansion.

**Comment by steveperry-53** [comment]
/approve
@sftim Will you give the lgtm when this looks ready.

**Comment by sftim** [comment]
I'm at SREcon this week so might not be around that often.

File: content/en/docs/concepts/storage/volumes.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/concepts/storage/volumes.md]: @houjun41544, do you agree with my comment? I recommend telling readers the name of the feature gate, and also adding a link to the page about feature gates. (That's how most similar pages handle it).

Comment (houjun41544) [reply content/en/docs/concepts/storage/volumes.md]: @sftim Sorry, I was out on vacation these days so I  didn't see your comment.
I agree with your suggestion.

File: content/en/docs/concepts/storage/volumes.md [hunk 2]
```diff
@@ -1158,7 +1158,7 @@ spec:
 
 
 Use the `subPathExpr` field to construct `subPath` directory names from Downward API environment variables.
-Before you use this feature, you must enable the `VolumeSubpathEnvExpansion` feature gate.
+This feature requires `VolumeSubpathEnvExpansion` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) to be enabled. It is enabled by default starting with Kubernetes 1.15.
```
Comment (sftim) [inline content/en/docs/concepts/storage/volumes.md]: nit:
```suggestion
This feature requires the `VolumeSubpathEnvExpansion` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) to be enabled. It is enabled by default starting with Kubernetes 1.15.
```
@houjun41544 do you agree?

**Pull request merged**